opsdroid/loader.py

```diff
@@ -588,6 +588,16 @@ def _is_gist_module(config):
         return "gist" in config
 
     def _install_module_dependencies(self, config):
+        """Install the dependencies of the module.
+        
```

Comment:
Same thing here, this line has white spaces could you remove them but keep the blank line?